Gopkg.lock

```diff
@@ -8,6 +8,14 @@
   pruneopts = "UT"
   revision = "70ad3c3262ada0444973f3f02e4a8339319236b3"
 
+[[projects]]
+  branch = "master"
+  digest = "1:e2a1ff1174d564ed4b75a62757f4a9081ed3b8c99ed17e47eb252b048b4ff018"
+  name = "github.com/asaskevich/EventBus"
```

Comment:
Should we add it to `Gopkg.toml` as a direct dependency?

Reply:
done